packages/node/src/schematics/application/application.spec.ts

```diff
@@ -244,6 +244,39 @@ describe('app', () => {
       );
     });
 
+    it.each`
```

Comment:
@mehrad-rafigh I don't think we need the `each` here 🤔

Reply:
I removed it.